segmenters/nlp/DeepSegmenter/README.md

```diff
@@ -1,3 +1,55 @@
 # DeepSegmenter
 
-Designed with ASR outputs in mind, DeepSegment uses BiLSTM + CRF for automatic sentence boundary detection. It significantly outperforms the standard libraries (spacy, nltk, corenlp ..) on imperfect text and performs similarly for perfectly punctuated text. 
\ No newline at end of file
+Designed with ASR outputs in mind, [DeepSegment](https://bpraneeth.com/projects/deepsegment) uses BiLSTM + CRF for automatic sentence boundary detection. It significantly outperforms the standard libraries (spacy, nltk, corenlp ..) on imperfect text and performs similarly for perfectly punctuated text. 
+
+Additional links:
+- [DeepSegment](https://bpraneeth.com/projects/deepsegment)
+- [BiLSTM-CRF paper](https://arxiv.org/abs/1508.01991)
+
+##Usage:
```

Comment:
Leave a space between the hashtags and the title for correct markdown :)

Reply:
✅